packages/sources/graphql/src/index.ts

```diff
@@ -2,4 +2,6 @@ import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute } from './adapter'
 import { makeConfig, NAME } from './config'
 
-export = { NAME, makeExecute, makeConfig, ...expose(NAME, makeExecute()) }
+const adapterContext = { name: NAME }
+
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute(), undefined, endpointSelector)
```